tests/Unit/Carriers/TextMapTest.php

```diff
@@ -23,7 +23,7 @@ public function testCreateATextMapWithItemsSuccess()
 
     public function testSettingAKeyValuePairSuccess()
     {
-        $textMap = TextMap::create();
+        $textMap = TextMap::fromMap();
```

Comment:
calling the function `fromMap` without an argument looks weird to me

Reply:
I just changed it, this case is only for the sake of testing `set`, ideally you create a `textMap` with values already.